site/content/en/docs/benchmarks/cpuUsage/linux/_index.md

```diff
@@ -0,0 +1,7 @@
+---
+title: "CPU Usage Benchmarks(Linux)"
+linkTitle: "CPU Usage Benchmarks(Linux)"
+weight: 1
+---
+
+![linux](/images/benchmarks/cpuUsage/linux.png)
```

Comment:
this image is missing

Reply:
Thank you for your pointing out!
I added the image.